BeeKit/Model/Goal.swift

```diff
@@ -62,6 +62,11 @@ public class Goal: NSManagedObject {
     @NSManaged public func addToRecentData(_ values: Set<DataPoint>)
     @objc(removeRecentData:)
     @NSManaged public func removeFromRecentData(_ values: Set<DataPoint>)
+    
+    @NSManaged public var dueByTable: DueByTable
```

Comment:
I suggest calling this variable `dueBy` - I've been trying to exactly match api names (modulo casing and underscores) so I don't need to remember both.